frontend/public/components/utils/details-item.tsx

```diff
@@ -36,14 +36,16 @@ export const DetailsItem: React.FC<DetailsItemProps> = ({
   defaultValue = '-',
   hideEmpty,
   children,
+  customDescription = undefined,
+  customDescriptionFooter = undefined,
 }) => {
   if (hideEmpty && _.isEmpty(_.get(obj, path))) {
     return null;
   }
 
   const reference: K8sResourceKindReference = referenceFor(obj);
   const model: K8sKind = modelFor(reference);
-  const description: string = getPropertyDescription(model, path);
+  const description: string = customDescription ?? getPropertyDescription(model, path);
```

Comment:
This doesn't seem like the right way to handle this. This component is specifically for getting API doc and has a lot of logic you don't need. I would just create the custom popover in the one place you need it. It's pretty trivial to add.